### PR TITLE
javaScript coercion

### PR DESCRIPTION
**Pull request by ThisIsAntony**
## JavaScript coercion

Add file index.html containing script with type coercion operations​

### CODE REVIEW

File: 1. Js_coercion_task/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        const obj = {};
+        const array = [];
+        const date = new Date(0);
+
+      console.log("=============JS type coercion explined============");
+
+      console.log("1. {} + [] + {} + [1] = ", obj + array + obj + [1]);
+      console.log("type - ", typeof (obj + array + obj + [1]))
+      console.log("2. new Date(0) + 1 = ", date + 1);
+      console.log("type - ", typeof (date + 1))
```
Comment (kaseopea) [inline 1. Js_coercion_task/index.html:20]: Semicolon is absent

File: 1. Js_coercion_task/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      console.log("3. true + false = ", true + false);
+      console.log("type - ", typeof (true + false))
```
Comment (kaseopea) [inline 1. Js_coercion_task/index.html:22]: Semicolon is absent

File: 1. Js_coercion_task/index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      console.log("4. null == '' = ", null == "");
+      console.log("type - ", typeof (null == ""))
```
Comment (kaseopea) [inline 1. Js_coercion_task/index.html:24]: Semicolon is absent

File: 1. Js_coercion_task/index.html (hunk 1 above)
Comment (ThisIsAntony) [reply 1. Js_coercion_task/index.html:20]: done.

File: 1. Js_coercion_task/index.html (hunk 2 above)
Comment (ThisIsAntony) [reply 1. Js_coercion_task/index.html:22]: done.

File: 1. Js_coercion_task/index.html (hunk 3 above)
Comment (ThisIsAntony) [reply 1. Js_coercion_task/index.html:24]: done.

**Pull request merged**